### DISCUSSION TITLE
How to make Svg image work?

### DISCUSSION THREAD

**Discussion by Marc-Antoine-Soucy** [unoplatform/uno#10750] [Q&A]
So, I tried using the [Documentation on using svg images](https://platform.uno/docs/articles/features/svg.html), and when I tried putting it in WinUI, it worked perfectly by just putting the svg in my assets and this code:
```
<Image Stretch="UniformToFill"
              Width="125"
               Height="125">
    <Image.Source >
        <SvgImageSource UriSource="ms-appx:///Assets/Images/circle_exclamation.svg" />
    </Image.Source>
</Image>
```
However, it didnt render on IOS and it threw an exception on android. So, I thought I would make a sample and open an issue, but the svg didnt show up either on WinUI/UWP on any of the samples I tried to make, is there some condition that must be met? (I looked at the windows [documentation ](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.imaging.svgimagesource?view=winrt-22621) but didnt find anything helpful.)

This is the error that was throwing on android.
```
Error		error: cannot find symbol
		if (getClass () == SvgCanvas.class) {
  symbol:   method getClass()
  location: class SvgCanvas	*******.Droid	C:\p\*****\***.Droid\obj\Debug\120\android\src\crc64309bc1dcfd7abb17\SvgCanvas.java	21	
Error		error: package crc641bba16f87fb10160 does not exist
	extends crc641bba16f87fb10160.SKSwapChainPanel	OutOfTruck.Droid	C:\p\*****\*****.Droid\obj\Debug\120\android\src\crc64309bc1dcfd7abb17\SvgCanvas.java	5	
```

**Comment by jeromelaban** [comment]
It looks like you need to include `SkiaSharp.Views.Uno.WinUI`. Is it explicitly specified in your project?

**Reply by Marc-Antoine-Soucy** [reply]
yes, I had both in my project (on platform other than windows)

**Reply by Marc-Antoine-Soucy** [reply]
![image](https://user-images.githubusercontent.com/90481654/208164288-737aa881-3435-4d63-8e33-6b9e6c86ed84.png)

**Reply by jeromelaban** [reply]
Thanks. This may otherwise mean a compilation error? Can you try deleting your bin/obj folder as well as uninstall the app on the device?

**Reply by Marc-Antoine-Soucy** [reply]
thanks for the help, I tried to do it again, but I still get the same error after deleting bin & obj and uninstalling the app

**Reply by jeromelaban** [reply]
@MartinZikmund does this look familiar?

**Reply by MartinZikmund** [reply]
Not immediately, @Marc-Antoine-Soucy can you please post an issue with the non-working repro (that is crashing on iOS and Android)? For WinUI the problem may be the limited rendering capabilities for SVG there, but I can check that too while I am looking into the other targets

**Reply by Marc-Antoine-Soucy** [reply]
https://github.com/unoplatform/uno/issues/10766
